lib/main.coffee

```diff
@@ -54,6 +54,14 @@ module.exports =
       items:
         type: 'boolean'
       order: 7
+    hidechars:
+      title: 'Hide character count'
+      description: 'Hides the character count from the view'
+      type: 'boolean'
+      default: 'false'
+      items:
+        type: 'boolean'
```

Comment:
Not sure what that `items` key is for. Maybe it's not needed for a single option key. Could you clarify this?